arch/arm64/kernel/cpuinfo.c

```diff
@@ -116,10 +119,9 @@ static int c_show(struct seq_file *m, void *v)
 #ifdef CONFIG_SMP
 		seq_printf(m, "processor\t: %d\n", i);
 #endif
-
-		seq_printf(m, "BogoMIPS\t: %lu.%02lu\n",
-			   loops_per_jiffy / (500000UL/HZ),
-			   loops_per_jiffy / (5000UL/HZ) % 100);
```

Comment:
Shouldn't be removed due to 4c512dd2edb28e10681beab01148ce56a230eb6e

Reply:
Hmm

Reply:
that matches msm-4.9